ntbs-service/Pages/ContactDetails/Edit.cshtml

```diff
@@ -0,0 +1,134 @@
+﻿@page "{handler?}"
+@using ntbs_service.Models
+@model ntbs_service.Pages.ContactDetails.EditModel
+@using static NHSUK.FrontEndLibrary.TagHelpers.FormGroupType
+
+@{
+    Layout = "Shared/_Layout";
+}
+
+<nhs-width-container container-width="Standard">
+    <h2 class="contact-details-fullname"> @Model.ContactDetails.FullName </h2>
+    <div class="contact-details-edit-container">
+        <h3 class="contact-details-section-heading"> Contact details </h3>
+    </div>
+    <div class="contact-details-container">
+        <form method="post" autocomplete="off">
+        
+            <input type="hidden" asp-for="ContactDetails.Username" />
+            <input type="hidden" asp-for="ContactDetails.GivenName" />
+            <input type="hidden" asp-for="ContactDetails.FamilyName" />
+            <input type="hidden" asp-for="ContactDetails.DisplayName" />
+            <input type="hidden" asp-for="ContactDetails.AdGroups" />
+            <input type="hidden" asp-for="ContactDetails.IsActive" />
+            <input type="hidden" asp-for="ContactDetails.IsCaseManager" />
```

Comment:
Hmmm, the fact that these points out that the POST route actually allows for these to be changed... I wonder if it'd be cleaner if it only allowed for the contact details themselves to be modified (and e.g. the repository method to not save the entire user model, but this selected number of fields..?)